rust/src/common.rs

```diff
@@ -14,6 +14,24 @@ macro_rules! take_until_and_consume (
   );
 );
 
+
+#[cfg(not(feature = "debug-validate"))]
+#[macro_export]
+macro_rules! debug_validate_bug_on (
+  ($item:expr) => {};
+);
+
+
+#[cfg(feature = "debug-validate")]
+#[macro_export]
+macro_rules! debug_validate_bug_on (
```

Comment:
indent looks off

Reply:
rustfmt does not seem to change anything here. What do you think should be changed?

Reply:
I see 2 space indent instead the expected 4

Reply:
I see the same indent in other macro as well `take_until_and_consume`. Should I change for both?

Reply:
Rustfmt does not format inside macros.